.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run test -- --watchAll=false
```

Comment:
create a new script called: npm run test:ci

Reply:
fixed it